### PR TITLE
add ExpansionHunter v5.0.0 image

### PR DESCRIPTION
**Pull request by hopedisastro**

### CODE REVIEW

**Comment by illusional** [comment]
Hey @hopedisastro, thanks for creating the PR! The hadolint linter has highlighted a few minor issues in your Dockerfile. Could you address these please?

File: images/expansionhunter/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    make \
+    pkg-config \
+    wget \
+    unzip \
+    zlib1g-dev \
+    libcurl4-openssl-dev \
+    libssl-dev
+
+# Install and configure conda
+RUN wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-${CONDA_RELEASE}-Linux-x86_64.sh -O /tmp/miniconda.sh && \
```
Comment (illusional) [inline images/expansionhunter/Dockerfile:31]: Is there any specific reason for installing conda? The packages you install below are from `pip`, and could probably save a bit of space by not including these.

File: images/expansionhunter/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+ARG CONDA_CMD=${CONDA_BIN}/conda
+ENV PATH=${CONDA_BIN}:$PATH
+ENV EH_VERSION=v5.0.0
+
+RUN apt-get update && apt-get install --no-install-recommends -qqy \
+    python3-dev \
+    python3-pip \
+    python \
+    python-dev \
+    awscli \
```
Comment (illusional) [inline images/expansionhunter/Dockerfile:17]: Is the `awscli` needed for running code through expansion hunter, or would it be safe to remove?

**Review by illusional — CHANGES_REQUESTED** [review]

File: images/expansionhunter/Dockerfile (hunk 1 above)
Comment (hopedisastro) [reply images/expansionhunter/Dockerfile]: No, I thought there was some relationship between pip and conda. I have removed conda.

File: images/expansionhunter/Dockerfile (hunk 2 above)
Comment (hopedisastro) [reply images/expansionhunter/Dockerfile]: I have removed.

File: images/expansionhunter/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,41 @@
+FROM ubuntu:20.04
+ARG DEBIAN_FRONTEND=noninteractive
+ENV PATH="/root/miniconda3/bin:${PATH}"
+ARG CONDA_RELEASE="4.6.14"
+ARG CONDA_INSTALL_DIR="/opt/conda"
+ARG CONDA_BIN=${CONDA_INSTALL_DIR}/bin
+ARG CONDA_CMD=${CONDA_BIN}/conda
+ENV PATH=${CONDA_BIN}:$PATH
```
Comment (illusional) [inline images/expansionhunter/Dockerfile:8]: You can remove these lines now that conda was removed.
```suggestion
```

File: images/expansionhunter/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+RUN wget --progress=dot:giga https://github.com/Illumina/ExpansionHunter/releases/download/${EH_VERSION}/ExpansionHunter-${EH_VERSION}-linux_x86_64.tar.gz && \
+    tar xzf ExpansionHunter-${EH_VERSION}-linux_x86_64.tar.gz && \
+    rm ExpansionHunter-${EH_VERSION}-linux_x86_64.tar.gz && \
+    mv /ExpansionHunter-${EH_VERSION}-linux_x86_64 /ExpansionHunter
+ENV PATH="/ExpansionHunter/bin/:$PATH"
+
+# Clean up
+RUN rm -rf /tmp/* \
+    /var/tmp/* \
+    /var/cache/apt/* \
```
Comment (illusional) [inline images/expansionhunter/Dockerfile:41]:
```suggestion
    /var/cache/apt/*
```

**Review by illusional — APPROVED** [review]
Great work, this Dockerfile looks good to me!
I've left two small suggestions (based on conda being removed).

**Pull request merged**